cypress/e2e/gui/basic.cy.js

```diff
@@ -0,0 +1,96 @@
+describe('Basic', () => {
+  beforeEach(() => {
+    Cypress.on('window:before:load', window => {
+      window.document.cookie = 'cookieConsent=accepted'
+    })
+    cy.visit('/')
+  })
+
+  it('keeps the filters when coming back from the customer details view', () => {
+    cy.get('[data-testid="size-filter"]').select('Small')
+    cy.get('[data-testid="industry-filter"]').select('Finance')
+    cy.contains('button', 'View').click()
+    cy.contains('h2', 'Customer Details').should('be.visible')
+    cy.contains('button', 'Back').click()
+    cy.contains('h1', 'EngageSphere').should('be.visible')
+    cy.get('[data-testid="size-filter"]').should('have.value', 'Small')
+    cy.get('[data-testid="industry-filter"]').should('have.value', 'Finance')
+  })
+  it('goes back to the customers list when clicking the "Back" button', () => {
+    cy.contains('button', 'View').first().click()
```

Comment:
```suggestion
    cy.contains('button', 'View').click()
```

Diferente do `cy.get()`, o `cy.contains()` só retorna o primeiro elemento encontrado, portanto, o `.first()` não é necessário aqui.

Escrevi sobre isso nos seguintes conteúdos:

- [Diferenças entre os comandos cy.get() e cy.contains() do Cypress](https://talkingabouttesting.com/2023/02/01/diferencas-entre-os-comandos-cy-get-e-cy-contains-do-cypress/)
- [cy.get() vs. cy.contains()](https://talkingabouttesting.com/2022/08/08/cy-get-vs-cy-contains/)